eng/Versions.props

```diff
@@ -139,6 +139,8 @@
     <MicrosoftDotNetBuildTasksFeedPackage>Microsoft.DotNet.Build.Tasks.Feed</MicrosoftDotNetBuildTasksFeedPackage>
     <MicrosoftNETCoreTargetsPackage>Microsoft.NETCore.Targets</MicrosoftNETCoreTargetsPackage>
     <MicrosoftNETCoreRuntimeCoreCLRPackage>Microsoft.NETCore.Runtime.CoreCLR</MicrosoftNETCoreRuntimeCoreCLRPackage>
+    <!-- XmlDocFileRoot needs to be defined here since we use it in packaging.props and docs.targets. It also uses Arcade defined property NuGetPackageRoot, this is the only codepath shared in between packaging.props and docs.targets with arcade properties defined. -->
+    <XmlDocFileRoot>$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', '$(MicrosoftPrivateIntellisensePackage)', '$(MicrosoftPrivateIntellisenseVersion)', 'IntellisenseFiles', 'net'))</XmlDocFileRoot>
```

Comment:
Breaking this comment in multilines to avoid scrolling in the metadata.
```suggestion
    <!-- XmlDocFileRoot needs to be defined here since we use it in packaging.props and docs.targets.
    It also uses Arcade defined property NuGetPackageRoot, this is the only codepath shared in between
    packaging.props and docs.targets with arcade properties defined. 
    -->
    <XmlDocFileRoot>$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', '$(MicrosoftPrivateIntellisensePackage)', '$(MicrosoftPrivateIntellisenseVersion)', 'IntellisenseFiles', 'net'))</XmlDocFileRoot>
```